common/djangoapps/student/tests/test_refunds.py

```diff
@@ -195,17 +193,3 @@ def test_multiple_refunds_dashbaord_page_error(self):
         self.client.login(username=self.user.username, password=self.USER_PASSWORD)
         resp = self.client.post(reverse('student.views.dashboard', args=[]))
         self.assertEqual(resp.status_code, 200)
```

Comment:
It turns out these tests were not actually testing anything. The call they were trying to test being broken is not actually being called in this code path and the checks in mock_order_endpoint to ensure that a call was being made were only passing because the httpretty data wasn't being cleaned up from previous tests.